playwright.config.ts

```diff
@@ -20,7 +20,7 @@ export default defineConfig<PluginOptions>({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 2 : 1,
```

Comment:
Do we need this change?

Reply:
Hmm yeah, it might hide flake, I'll remove it for now.